cloudwatch.tf

```diff
@@ -0,0 +1,9 @@
+resource "aws_cloudwatch_log_resource_policy" "route53_query_logging_policy" {
+  policy_document = "${data.aws_iam_policy_document.route53_query_logs.json}"
+  policy_name     = "route53-query-logging-policy"
+}
+
+resource "aws_cloudwatch_log_group" "query_log" {
+  name              = "/aws/route53/${var.hosted_zone_id}"
```

Comment:
Is there a reason that we are using the hosted_zone_id and not the name of the hosted zone to name the log group? I think that the latter would be easier to read...